### PR TITLE
Quote details table

### PR DESCRIPTION
**Pull request by sampettersson**
This PR extends giraffe with a cross schema extension that on a quote builds a table with labels and values that allows the app to easily render it without digging through quote details, this will be used to drive this component in the apps:

Screenshot 2021-04-21 at 11 27 04

### CODE REVIEW

File: src/translations/da_DK.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (hedvigoscar) [inline src/translations/da_DK.json:1]: Should we gitignore these and download them in CI? 🤔 single source of truth and all

**Review by hedvigoscar — APPROVED** [review]
I like it!

File: src/translations/da_DK.json (hunk 1 above)
Comment (sampettersson) [reply src/translations/da_DK.json:1]: @hedvigoscar web-onboarding did it like this, not really sure how we should approach it 🤔 it's kinda nice to have them checked in to not have to fiddle too much imo

But we can change it later ofc.

**Review by hugokallstrom — COMMENTED** [review]
Looks good! Could we take 5-10 minutes after standup tomorrow and go through this? Not 100% on how this is connected to other services.

**Comment by sampettersson** [comment]
@hugokallstrom absolutely :)

**Pull request merged**